test/upgrade/cert-manager-detection/00-assert.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
soft opinion: this is a pretty heavy test basically making sure that we choose `v1beta1` in the old cert-manager version and `v1` in the newer one. A unit-test would be better. Otherwise, we're basically testing the infrastructure. Which is something we should try to avoid. But then again - cert-manager is an important part of the infra so maybe it is justified.